test/cpp/end2end/rls_end2end_test.cc

```diff
@@ -158,7 +159,10 @@ class FakeResolverResponseGeneratorWrapper {
 class RlsEnd2endTest : public ::testing::Test {
  protected:
   static void SetUpTestSuite() {
-    GPR_GLOBAL_CONFIG_SET(grpc_client_channel_backup_poll_interval_ms, 1);
+    grpc_core::SetEnv("GRPC_EXPERIMENTAL_ENABLE_RLS_LB_POLICY", "true");
```

Comment:
I don't think this is needed.  We removed this env var long ago.  (I assume this is left over from some older code?)

Reply:
looks like it yeah! (gosh this has been backing for a while)